deploy/helm/templates/daemonset.yaml

```diff
@@ -157,4 +162,11 @@ spec:
       tolerations:
 {{ toYaml .Values.ndm.tolerations | indent 8 }}
 {{- end }}
+{{- if .Values.ndm.securityContext }}
+      securityContext:
+{{ toYaml .Values.ndm.securityContext | indent 8 }}
+{{- end }}
+{{- if .Values.ndm.hostNetwork }}
+      hostNetwork: true
```

Comment:
@sonasingh46  Why was this change required? This is mandatory for NDM to work. If users disable it, we wont receive any udev events.

Reply:
changed